tests/common/mod.rs

```diff
@@ -265,7 +264,7 @@ pub async fn kill_container(docker: &Docker, container_name: &'static str) -> Re
 }
 
 #[allow(dead_code)]
-pub async fn create_image_hello_world(docker: &Docker) -> Result<(), Error> {
+pub async fn create_image_hello_world(docker: &Docker) -> Result<String, Error> {
```

Comment:
Shouldn't `create_image_hello_world` return a string so that tests know what image to fiddle with in the test?